From randy.dunlap@oracle.com  Tue Mar 10 21:07:42 2009
From: Randy Dunlap <randy.dunlap@oracle.com>
Date: Mon, 02 Mar 2009 09:54:57 -0800
Subject: Staging: line6: depends on SND
To: Stephen Rothwell <sfr@canb.auug.org.au>
Message-ID: <49AC1D71.4050303@oracle.com>

From: Randy Dunlap <randy.dunlap@oracle.com>

line6 code has lots of dependencies on ALSA (and build errors),
so express that in Kconfig.

Signed-off-by: Randy Dunlap <randy.dunlap@oracle.com>
Cc: Markus Grabner <grabner@icg.tugraz.at>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/staging/line6/Kconfig |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/staging/line6/Kconfig
+++ b/drivers/staging/line6/Kconfig
@@ -1,6 +1,6 @@
 config LINE6_USB
 	tristate "Line6 USB support"
-	depends on USB
+	depends on USB && SND
 	help
 	  This is a driver for the guitar amp, cab, and effects modeller
 	  PODxt Pro by Line6 (and similar devices), supporting the
